### DISCUSSION TITLE
Need help on the json which have the key:value pairs

### DISCUSSION THREAD

**Discussion by kittu444** [json-schema-org/community#292] [General]
Hi,
I have a json which have the key : value pairs which trying to convert to JSON schema. Can you help me how can I have the values dynamically for both key:value?   [ i.e: test, testa, testb, testc, testd :- needs to be dynamic ]

{ 
  "test" : {
  "content":{
		"testa"  : "testb",
		"testc" : "testd"
	}
   }
}
Thank you.

**Comment by gregsdennis** [comment]
Thanks for the question.

I see you've also asked on [StackOverflow](https://stackoverflow.com/questions/75078068).  I think that forum is probably a better place for this, and I'll just leave the link here.
